src/main/java/uk/gov/hmcts/reform/cpo/service/impl/CasePaymentOrdersServiceImpl.java

```diff
@@ -111,6 +108,14 @@ private PageRequest getPageRequest(CasePaymentOrderQueryFilter casePaymentOrderQ
         );
     }
 
+    private String getUserId() {
+        try {
+            return securityUtils.getUserInfo().getUid();
+        } catch (Exception e) {
+            throw new IdAMIdCannotBeRetrievedException(IDAM_ID_NOT_FOUND);
```

Comment:
```suggestion
            throw new IdAMIdCannotBeRetrievedException(IDAM_ID_RETRIEVE_ERROR);
```

Reply:
Resolved in 6760f7ddf235c1c4bdef814215dc87ff7a6aa96f